Documentation update

sam-brett opened the pull request.

sam-brett committed:
refactoring field name

sam-brett committed:
refactoring field name

sam-brett committed:
adding functionality

sam-brett committed:
adding tests

sam-brett committed:
Merge remote-tracking branch 'origin/develop' into develop

# Conflicts:
#	src/main/java/uk/co/ramp/event/ContactEventProcessor.java
#	src/main/java/uk/co/ramp/people/Case.java
#	src/main/java/uk/co/ramp/people/Human.java
#	src/main/java/uk/co/ramp/people/PopulationGenerator.java
#	src/test/java/uk/co/ramp/event/ContactEventProcessorTest.java

sam-brett committed:
Updating first few sections.

sam-brett committed:
Adding basic info on Isolation Policies

sam-brett commented:
@vinopm 

I've just added the sections about Isolation Policy. If you could have a look when you get a chance. I'll be working on the Tracing bit next

sam-brett committed:
Correcting files that shouldn't have been checked in

vinopm commented on input/homogeneous_contacts.csv:
```diff
@@ -20,21 +20,21 @@ time,from,to,weight,label
 0,594,625,3.870173412,label
-0,96,632,5.811327709,label
 0,322,636,14.82281342,label
+0,320,800,3.270327909,label
```
The changes in this file should be excluded?

vinopm replied:
I'm not sure if there was a reason to include this

vinopm commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,20 +145,63 @@ go on in timesteps.
+
+## AgeData
+
+The age data file contains a list of id and age. This has been added 
+to allows the user to use age graduated contact files. Should the file 
+be longer than the population, additional input will be ignored. Should 
+it be shorter, the ages will be generated as per the population settings file. 
```
I think it's not really about how long the age file input is... I think the distinction is more to do with whether or not the id exists in the age data file. If the id exists we will use the provided age, otherwise we would generate it as per the population settings file

vinopm commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,20 +145,63 @@ go on in timesteps.
+### Default Policy
+
+The default policy allows a global baseline to be set. For example, 
+in an older population 10% of the population may shield.
```
We do not have the versatility yet to condition the policy on the individuals age, but definitely can be added in future with the current framework

vinopm commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,20 +145,63 @@ go on in timesteps.
+
+### Alert Policy
+
+The alert policy defines the isolation policy for a person in any state. 
+These follow the same rules as the Virus Policy.
```
Probably worth expanding on this and explaining that these policies are dependent on the alert status of individuals (with a list of the alert status options for clarity)

vinopm reviewed (dismissed):
On the whole looks good to me, nice work!

sam-brett committed:
Adding documentation for Alert Events and Tracing

sam-brett committed:
Removing

sam-brett committed:
Correcting files that shouldn't have been checked in

sam-brett committed:
Update Build_documentation.md

Adding HTML tags to reformat some of the images that are too large

sam-brett committed:
Adding office files

sam-brett committed:
Merge remote-tracking branch 'origin/documentationUpdate' into documentationUpdate

sam-brett committed:
Merge branch 'develop' into documentationUpdate

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -13,19 +13,22 @@ This model using a
 SE<sub>1</sub>E<sub>2</sub>I<sub>a</sub>I<sub>s</sub>RD model with the
```
Change typo on line 12 from 'This model using a...' to 'This model uses a...'

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -13,19 +13,22 @@ This model using a
-I<sub>s</sub> – Infected and symptomatic, will test positive
+Sev – Infected and severely symptomatic, will test positive 
 
 R – Recovered
```
Add 'D - Dead'

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -47,11 +58,11 @@ An example can be shown in Figure 2. The to and from fields are the ids
 of the nodes in contact. The weight field is a measure of the duration,
 intensity or proximity of the contact. This is dimensionless number and
```
Suggested change:
The contact data is a CSV file containing the temporal network of contacts between individuals in a population.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -39,6 +42,14 @@ this document.
 
+### InputLocations.json
+
+The first input file is the inputLocations.json file that allows all the other inputs to be configured. 
+This allows the user to have multiple configurations defined. File locations are relative to the input folder.
```
I think this needs to be a little clearer. Perhaps the following:

```suggestion
The _InputLocations.json_ file allows the user to specify the location of all other inputs files relative to the _/input_ folder. This allows the user to organise multiple simulation configurations.
```

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -47,11 +58,11 @@ An example can be shown in Figure 2. The to and from fields are the ids
+field is to enable filtering of contacts based on their label.
+
+<img src="image2.png" alt="Contacts File" width="500">
 
```
I'm not quite sure what you intend to say in line 60f., something like "This is dimensionless number and is used as threshold for determining the spread of infection, allowing to filter the individual contacts regarding whether they should be alerted or not."?

Also I suggest to change line 61 to "The contacts data contains an additional field 'label' defining where the contact occurred (e.g. school, work, home, ..). This 'label' field can be used to filter contacts, i.e. to dynamically switch them on or off at the beginning of the simulation."

sam-brett replied:
I've made the first change, but I don't think we ended up implementing the label filtering in the end. Unless it happened when I was away.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -13,19 +13,22 @@ This model using a
 SE<sub>1</sub>E<sub>2</sub>I<sub>a</sub>I<sub>s</sub>RD model with the
 following transitions allowed:
```
A few things to change. Perhaps:

```suggestion
This model uses a S-E-Asym-Pre-Sym-Sev-R-D model that allows the following transitions:
```

If the above suggestion is used, be sure to remove line 12 of the un-edited version.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -80,10 +91,18 @@ from outside the network interacting. This is the probability per
 Figure 3. The diseaseSettings.json file
 
+
+The exposureProbability4UnitContact (expUnitContact) and exposureExponent (exp) values are 
+used to determine the chance of infection for a given contact. 
```
I think the mathematical implementation may be worth explicitly declaring.
Also the name _exposureProbability4UnitContact_ sounds too similar to the Java unit testing module. Perhaps a more descriptive name should be added at some point.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -39,6 +42,14 @@ this document.
+
 ### Contacts.csv
 
 The contact data is a CSV file contain network data that shows contacts
```
```suggestion
The contact data is expected to be a CSV file containing network data that shows contacts
```

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
 Infected: the number of the population initially infected at t=0. Must
 be greater than 0.
 
```
Are the 'initialExposures' now called 'Infected'? If not I'd change this to the corresponding variable name (or add in brackets 'initialExposures').

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
 has been processed, the steady state flag enables the active cases to
 reach a resolution (recovered or dead) with no further random infections
 occurring.
 
```
Suggested additions (or has this changed in the input file?):
- "timeStepsPerDay": allows to set the timestep (per day) for disease progression e.g. half days ('2') or quarter days ('4').
- "timeStepSpread": set probability for spreading the disease in each chosen timestep, e.g. for half days [0.4,0.6] or quarter days [0.2, 0.2, 0.4, 0.2].

Can we add an example for the command line argument, e.g 'gradle run --args='--overrideInputFolderLocation=input/scenarios/scenario1a_tracinglevel1 --overrideOutputFolderLocation=output0 --seed=3'

sam-brett replied:
I think the gradle command example should be added to the readme

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
 
+
+## AgeData
+
```
I suggest renaming AgeData to PopulationData, or IDdata, or PersonData as this contains the IDs of the age-structured individuals (i.e. a list of ID's and associated age).

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
-Seed: the seed for the random number generator. Will be modified by the
-command line argument if included. This ensure a reproducible output.
-
 Steady state: if there are still active infections when the contact data
 has been processed, the steady state flag enables the active cases to
 reach a resolution (recovered or dead) with no further random infections
 occurring.
```
Is this parameter used?

sam-brett replied:
removed

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
+
+## Isolation Policies
+
+The isolation policy can be split into 3 sections.
```
```suggestion
An isolation policy can be split into 3 sections.
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
+
+### Alert Policy
+
+The alert policy defines the isolation policy for a person in any state. 
```
```suggestion
The alert policy defines the isolation policy for a person in any alert state. 
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -324,9 +407,10 @@ section fills in the details on how this section works. Figure 10 shows
-are randomly sampled and set to EXPOSED to begin the simulation.
+Upon entering the method, a set of infected cases are read from the initial 
+exposures file. If this file is shorter than the value specified in 
+RunSettings, they will be filled to the correct length. Likewise, if it is too 
```
How are they filled?

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
 
-Running the propagation of infection is the heart of the calculation.
-Figure 12 shows the flow of data.
+The runContactData method within Outbreak.java is where the main calculations in the code are triggered. 
```
```suggestion
The runContactData method within Outbreak.java is where the main calculations are triggered. 
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+
+The events within the model are handled by a group of lists that contain specific events that can occur in the model. These are:
+
+1.	Alert Event
+2.	Contact Event
+3.	Infection Event
+4.	Virus Event
```
Since this is a numbered list, perhaps the order should reflect the order they are called in the code.

```suggestion
1.	Virus Event
2.	Infection Event
3.	Contact Event
4.	Alert Event
```

sibylleM replied:
I agree with Ed on the order...

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -121,43 +146,97 @@ go on in timesteps.
+
+The default policy allows a global baseline to be set. For example, 
+10% of the population may be shielding by default.
+
```
It's not clear to me what the default policy is and where it is set. Also, I would match the example (% shielding) to the picture.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+
+### Alert Event
+
+The alert event processor is used to cycle through the states of disease alert that a member of the population can be in at any one point. 
```
Is it worth mentioning the other event processor classes in the other sections?

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
-interest. Before evaluating the contact further, the two cases are
-checked to see if their virus statuses are different, if they are the
-contact is evaluated.
+The person has their status set to a corresponding status and the next status is calculated for a time in the future. 
```
Which status? Alert status or Virus status?

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
-checked to see if their virus statuses are different, if they are the
-contact is evaluated.
+The person has their status set to a corresponding status and the next status is calculated for a time in the future. 
+The result of the test is determined by whether the individual is currently infectious and a dice roll against the accuracy 
+of the test. The number of false positives and negatives is recorded as an output statistic. 
```
```suggestion
The result of a test is determined by whether the individual is currently infectious and a dice roll against the accuracy of a test. The number of false positives and negatives is recorded as an output statistic. 
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
-PersonA is the more severe case; the least severe case is personB. A
-dangerous mix is when personA is infectious (E2, I, IA) and personB is
-susceptible. If this is the case, the following test is conducted:
+An infection event is triggered in three ways:
```
```suggestion
An infection event can be triggered in three ways:
```

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -205,10 +284,10 @@ should these resources not be loaded, so the code may exit here. A log
+- **overrideOutputFolderLocation:** allows a different output folder to be assigned. Defaults to /output.
 
 The input files are described in Inputs.
 
```
Aha, sorry, I had asked for a command line example somewhere above but maybe it fits better here?

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+An infection event is triggered in three ways:
 
-![](eq6.gif)
+1. The id has been chosen as a seed infection.
+2. The id has been randomly chosen to be a seed.
+3. The id has been in contact with an infectious individual and has subsequently contracted the infection. 
```
Replace 'id' with 'individual'

sibylleM replied:
Just to add to Ed, maybe worthwhile using 'individual (id)'

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+3. The id has been in contact with an infectious individual and has subsequently contracted the infection. 
 
-Where n is a random number.
+Once a SUSCEPTIBLE individual is exposed to the infection, an infection event is triggered which in turn creates a virus event. 
```
Is this still true given the order of the event processing in EventRunnerImpl.java has been reversed?

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+3. The id has been in contact with an infectious individual and has subsequently contracted the infection. 
 
-Where n is a random number.
+Once a SUSCEPTIBLE individual is exposed to the infection, an infection event is triggered which in turn creates a virus event. 
+The infection event occurs one time step after the contact. 
```
Is this still true given the order of the event processing in EventRunnerImpl.java has been reversed?

See my suggested edit under the Event Runner section.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+### Virus Event
 
-Figure 12. The logic flow for reviewing contact data
+A virus event is the progression of the virus through in compartments. The initial infection is not included as this is 
```
```suggestion
A virus event is the progression of the virus through it's compartments. The initial infection is not included as this is 
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+A virus event is the progression of the virus through in compartments. The initial infection is not included as this is 
+an Infection Event. The processing of the event is very similar to the previous sections, with the exception of checking 
+for alerts based on the virus status of the individual. This process creates alert events for people who have been in 
+contact with the individual. The individual has a reportingCompliance field that may reduce the chance of an individual
```
```suggestion
contact with an infectious individual. The individual has a reportingCompliance field that may reduce the chance of an individual
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+### Virus Event
 
-Figure 12. The logic flow for reviewing contact data
+A virus event is the progression of the virus through in compartments. The initial infection is not included as this is 
+an Infection Event. The processing of the event is very similar to the previous sections, with the exception of checking 
+for alerts based on the virus status of the individual. This process creates alert events for people who have been in 
+contact with the individual. The individual has a reportingCompliance field that may reduce the chance of an individual
+reporting symptoms. The alert checker and tracing policies are covered below.  
```
```suggestion
A virus event is the progression of the virus through intscompartments. The initial infection is not included as this is 
an infection event. The processing of the event is very similar to the previous sections, with the exception of checking 
for alerts based on the virus status of an individual. This process creates alert events for people who have been in 
contact with an infectious individual. An individual has a reportingCompliance field that may reduce the chance of reporting symptoms. The alert checker and tracing policies are covered below.  
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -381,97 +465,103 @@ Figure 11. Infection map example
+### 	Determining Isolation Policy
 
-#### Virus Status
+The isolation status of all individuals is stored in a map. These are used to determine if an individual has already 
```
Is the fact that the data is stored in a map required?

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -324,9 +407,10 @@ section fills in the details on how this section works. Figure 10 shows
+exposures file. If this file is shorter than the value specified in 
+RunSettings, they will be filled to the correct length. Likewise, if it is too 
+long, it is truncated to the correct length.
 
```
Suggested edits, perhaps:
Upon entering the method, a set of initial infections (index cases) are read from the initial exposures file ('initialExposures.csv'). If this file is shorter than the value of initially infecteds specified in RunSettings.json, they will be filled to the required length by randomly infecting individuals in the population.

sibylleM replied:
I used 'index cases' but you use 'seed' somewhere below, so perhaps we should stick with seed or leave 'index cases' out.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -294,9 +378,8 @@ simulation.
-The outbreak class will be handled in much more detail later on in
-\<\<Section XYZ\>\>. This section will focus purely on the
-initialisation.
+The outbreak class will be handled in much more detail later on. 
```
If this is true, it's not explicitly clear that it has been covered. However, I may have just missed it.

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -103,10 +122,16 @@ Gender balance: this is the ratio of men:women, so there are 99 men for
 
+### Initial Exposures
+
+The initial exposures file contains a list of IDs of the people who are in the
+exposed category at t=0. If this list is longer than the initial exposures, it 
+will be truncated. If it is too short, it will be filled randomly. 
```
```suggestion
The initial exposures file contains a list of IDs of the individuals who are
initially exposed at the start of the simulation, i.e. at the first time step. If this list is longer than the initial exposures, it will be truncated. If it is too short, it will be filled randomly. 
```

EdTownsend1 commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -47,11 +58,11 @@ An example can be shown in Figure 2. The to and from fields are the ids
 of the nodes in contact. The weight field is a measure of the duration,
```
What is a node in the contact network?

EdTownsend1 reviewed (commented):
I've only made comments on the _Build Documentation.md_ file. However, I have put some of my comments about some of the other files.

- In _AlertEvent.png_ one of the decision boxes contain '== current status'. Is this a typo?
- I'm assuming that the _Build Documentation.docx_ file is changed in sync with the corresponding _.md_ file, therefore I've only commented on the _.md_ file.

sibylleM commented on docs/doc_v1.0/Build_documentation.md:
```diff
@@ -276,12 +354,18 @@ At present, health is randomly sampled from a uniform distribution. It
 be a very healthy individual, a score of 0 is a very frail individual.
 
-#### Compliance
+#### Isolation Compliance
```
For all of the compliance values, it would be good to add where they are set. For instance, I think some of them are set in the population generator.java, if I'm not mistaken.

sibylleM reviewed (commented):
Similar as Ed, I have made some comments / suggestions on the Build_documentation.md. Regarding Ed's comment on the doc file, I don't think we will need the doc file any longer (and a waste of time to convert everything), we should stick with the .md only.

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Update docs/doc_v1.0/Build_documentation.md

Co-authored-by: EdTownsend1 <67421899+EdTownsend1@users.noreply.github.com>

sam-brett committed:
Merge branch 'develop' into documentationUpdate

sam-brett committed:
Addressing review comments:
Removing steady state flag as it's no longer used. Updating StandardProperties documentation to reflect this and new day offset

sam-brett committed:
Addressing other review comments

sam-brett committed:
Updating build docs

EdTownsend1 committed:
Re-worded and tidied first four sections

Tidied the Introduction section.
Slightly re-worded the Inputs section.
Re-worded the InputLocation.json section first paragraph.
Added info about how the CTM behaves with missing entries.
Re-formatted the info in Contacts.csv section

EdTownsend1 committed:
Update Build_documentation.md

Fixed typo to inputLocations.json from InputLocations.json 
Re-worded and re-formatted diseaseSettings.json section

EdTownsend1 committed:
Removed image3.png for an updated image.

New image is called diseaseSettings.png

EdTownsend1 committed:
Updated diseaseSettings.png

Replaced image3.png

EdTownsend1 committed:
Updated the diseaseSettings.json example image

EdTownsend1 committed:
Deleted inputLocations.png

Replaced with updated version

EdTownsend1 committed:
Updated inputLocations.png

EdTownsend1 committed:
Delete inputLocations.png

EdTownsend1 committed:
Re-uploaded updated inputLocations.png

EdTownsend1 committed:
Update Build_documentation.md

Consistent formatting
Renamed section names to be independent of the file name.
Re-worded and re-formatted the Population Settings section.
Re-worded and re-formatted the Initial Exposures section.
Re-worded and re-formatted the Run Settings section.
Re-worded and re-formatted the Age Data section.

EdTownsend1 committed:
Edited typo in Age Data section

EdTownsend1 committed:
Fixed multiple typos.

EdTownsend1 committed:
Added additional section + reformatting

Started a Contents section
Added Input Files/Infection Rates section.
Added Input Files/Isolation Policies section.
Added Input Files/Tracing Policies section.
Started to re-word + re-format Isolation Policies/Virus Policy section
Added Isolation Policies/Global Policy section.

EdTownsend1 committed:
Added infectionRates example image

EdTownsend1 committed:
Updated runSettings example file.

EdTownsend1 committed:
Removed old runSettings example image.

EdTownsend1 committed:
Removed old populationSettings example image

EdTownsend1 committed:
Re-formatted section to confirm with data pipeline

EdTownsend1 committed:
Minor reformatting of Input Files section.

EdTownsend1 committed:
Removed out-of-date inputLocations example image

EdTownsend1 committed:
Added updated inputLocation image

EdTownsend1 committed:
Updates Input Files section for Data Pipeline API

EdTownsend1 committed:
Removed out-of-date diseaseSettings example image

EdTownsend1 committed:
Add three new images

New images are consistent with the Data Pipeline API implementation into the Contact Tracing Model

EdTownsend1 committed:
Scaled config file example image

EdTownsend1 committed:
Added Isolation Policy and Global Policy section

EdTownsend1 committed:
Added globalPoliciesExample image

EdTownsend1 committed:
Updated VirusPolicy + AlertPolicy section

EdTownsend1 committed:
Added virus+alert policy example images

EdTownsend1 committed:
Fixed image links

EdTownsend1 committed:
Updated multiple sections

Edited Tracing Policies section
Edited Default Policy section
Edited Virus + Alert Statuses sections.

EdTownsend1 committed:
Added new images

Added tracingPoliciesExample image
Added defaultPoliciesExample image

EdTownsend1 committed:
Minor edits

Fixed typos
Moved the Isolation Policies and Tracing Policies section into the Data Pipeline Input Files

EdTownsend1 committed:
Fixed images

sibylleM reviewed (dismissed):
Hi @sam-brett and @EdTownsend1, many thanks again for updating the build documentation. Looks good and overall happy with the changes. The only thing worth mentioning might be to add where the compliance parameters are set (and where they can be changed, I think the PopulationGenerator.java) and maybe stressing that the policies are local and not part of the dataAPI (though I can add these things later as well). As John pointed out, it might be easier to first merge all of the changes and if there is anything else I can make suggestions on a new branch.

EdTownsend1 committed:
Merge branch 'develop' into documentationUpdate

EdTownsend1 committed:
Merge branch 'develop' into documentationUpdate

EdTownsend1 reviewed (approved):
Sam to double check the .java changes then all good to merge on my end.
